shopinvader_delivery_carrier/services/cart.py

```diff
@@ -133,6 +133,13 @@ def _delete_item(self, cart, params):
         self._unset_carrier(cart)
         return res
 
+    def update(self, **params):
+        if params.get('shipping', {}).get('address'):
```

Comment:
@bguillot are we sure that we always receive all the information of the SO? Is that the expected behavior?

Reply:
yes it's the format for updating the shipping address
I have added a test.